app/src/main/java/org/simple/clinic/teleconsultlog/drugduration/DrugDurationValidationResult.kt

```diff
@@ -0,0 +1,10 @@
+package org.simple.clinic.teleconsultlog.drugduration
+
+import android.os.Parcelable
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
```

Comment:
We don't need to explicit parcel enums because they are `Serializable` by default.